Makefile

```diff
@@ -26,3 +26,32 @@ envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
+GOLANGCI ?= $(LOCALBIN)/golangci-lint
+
+.PHONY: golangci
+golangci: $(GOLANGCI) ## Download golangci-lint locally if necessary.
+$(GOLANGCI): $(LOCALBIN)
+	test -s $(LOCALBIN)golangci-lint || GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.2
+
+
+.PHONY: lint
+lint: golangci
+	golangci-lint run ./... --timeout 5m
+
+GOLANGCI ?= $(LOCALBIN)/golangci-lint
+
+.PHONY: golangci
+golangci: $(GOLANGCI) ## Download golangci-lint locally if necessary.
+$(GOLANGCI): $(LOCALBIN)
+	test -s $(LOCALBIN)golangci-lint || GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.2
+
+GCI ?= $(LOCALBIN)/gci
+
+.PHONY: formatter
+formatter: $(GCI) ## Download gci locally if necessary.
```

Comment:
I would prefer to avoid depending on an external tool in someone's personal repo unless it brings a lot of benefit. I dont think that import ordering is worth the exposure.

Reply:
Import order can be annoying in PR reviews and may cause merge conflicts. I prefer standardization but this change was mainly driven by the linter. We would need to remove the [`cgi`](https://github.com/substratusai/lingo/pull/62/files#diff-9917ddc9f1c3304218f7269265b746d997c5c0615478177b5fceecd33ef47cb5R29) rule

Reply:
I think CGI is fine, it has a lot of contributors and is used in a lot of projects: https://github.com/daixiang0/gci